src/lib/split-view/split-view-panel/split-view-panel.html

```diff
@@ -0,0 +1,11 @@
+<template>
+  <div class="forge-split-view-panel" id="root" part="root">
+    <div class="forge-split-view-panel__handle" id="handle" part="handle" role="separator" aria-controls="content" tabindex="0">
```

Comment:
We should ensure that this element gets focus when clicking with the mouse. Right now you can only tab to it, but the default action should to set focus (can we make that cancellable too?)

Another item about the handle might be to show some feedback when the user reaches the min or max size. I'm thinking we could trigger the ripple effect, or maybe flash the bar using the danger theme color (maybe both together?). Just something to tell them they've reached the limit and it's not a bug basically.